speechbrain/nnet/CNN.py

```diff
@@ -654,6 +657,179 @@ def _check_input(self, shape):
         return in_channels
 
 
+class ConvTranspose1d(nn.Module):
```

Comment:
Ok. this part should be removed and we should use the one just merged in the development branch